src/components/projectSelect/index.tsx

```diff
@@ -110,7 +110,9 @@ export const ProjectSelect: React.FC<ProjectSelectProps> = ({
           projects={projectGroup.projects}
           name={projectGroup.name}
           onClick={onClick}
-          repoIdentifier={getRepoId(projectGroup.projects)}
+          repoIdentifier={
+            isProjectSettingsPage && getRepoId(projectGroup.projects)
```

Comment:
did you test this by viewing it on the project settings page and making sure that the dropdown loads as expected?

Reply:
It does indeed still work as expected! Are there cypress tests testing this?

Reply:
there is one